server/backends/chunkstore/chunkstore.go

```diff
@@ -367,16 +369,19 @@ type WriteResult struct {
 	Err            error
 	Size           int
 	LastChunkIndex uint16
+	Timeout        bool
```

Comment:
Is this `Timeout` flag computable as `status.IsDeadlineExceeded(result.Err)` ?

Reply:
Nope, it tells you if the write is a flush due to timeout while waiting for enough bytes to hit the threshold to write to disk.